odoo/addons/base/models/ir_mail_server.py

```diff
@@ -201,7 +201,15 @@ def _get_test_email_addresses(self):
                 return self.from_filter, "noreply@odoo.com"
             # All emails will be sent from any address in the same domain
             default_from = self.env["ir.config_parameter"].sudo().get_param("mail.default.from", "odoo")
-            return f"{default_from}@{self.from_filter}", "noreply@odoo.com"
+            if "@" not in default_from:
```

Comment:
Maybe better to create a variable and return it at the end?
To avoid copy/pasting "noreply@odoo.com" 4 times.

Reply:
good idea